static/css/section/_server.scss

```diff
@@ -1,213 +1,98 @@
-/*	@section server
---------------------------------------------------------------
-    Contents:
-        @section general
-        @section server-services
--------------------------------------------------------------- */
-
-/*	@section general
--------------------------------------------------------------- */
-
-body.server {
-
-  .row-hero { /* First bit of content on a page, normally contains h1 and sometimes a bit of blurb */
+@charset 'UTF-8';
+
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Server section
+//  Author:			Web Team at Canonical Ltd
+//  Last edited by: Peter Mahnke
```

Comment:
Could we remove `Last edited` comments, as mentioned in  https://github.com/ubuntudesign/www.ubuntu.com/pull/870#r79427677